server/game/cards/13.1-AtG/DragonGate.js

```diff
@@ -0,0 +1,25 @@
+const DrawCard = require('../../drawcard.js');
+const TextHelper = require('../../TextHelper');
+
+class DragonGate extends DrawCard {
+    setupCardAbilities(ability) {
+        this.plotModifiers({
+            gold: 1
+        });
+        this.action({
+            title: 'Sacrifice to draw 2 cards',
+            cardCondition: card => card.attachments.length >= 2,
```

Comment:
Actually, that condition isn't correct either. It would miss attachments that you control that you've placed on opponent cards. You could use `this.game.getNumberOfCardsInPlay` instead and additionally check that the card is controlled by the current player.

Reply:
Got it. Thanks for feedback. I will fix it tomorrow because I hlave already plans for today.